deploy/Chart/values.yaml

```diff
@@ -21,6 +21,9 @@ global:
   #   url: "http://jaeger-collector.radius-monitoring.svc.cluster.local:9411/api/v2/spans"
   #
 
+  azureWorkloadIdentity:
```

Comment:
Would it make sense to add a comment here that it is disabled by default, hence the config value?